k9mail/src/main/java/com/fsck/k9/Preferences.java

```diff
@@ -32,7 +32,7 @@ public static synchronized Preferences getPreferences(Context context) {
 
 
     private Storage mStorage;
-    private Map<String, Account> accounts = null;
+    private HashMap<String, Account> accounts = null;
```

Comment:
This stops us from providing a HashMap that doesn't support `put()` (See `MessagingControllerTest`)